be2-scala/linter/input/src/main/scala/fix/StripMarginOnRegex.scala

```diff
@@ -0,0 +1,13 @@
+/*
+rule = StripMarginOnRegex
+ */
+package fix
+
+object StripMarginOnRegex {
+  def test() = {
+//    val regex = "match|this".stripMargin.r // assert: StripMarginOnRegex
+//    "match|that".stripMargin.r // assert: StripMarginOnRegex
```

Comment:
If not necessary anymore should be removed. In addition, your test should have at least a match checking that your rule is well triggered